src/swagger/api.yaml

```diff
@@ -698,7 +698,6 @@ paths:
           type: string
         - in: body
           name: applicationGroupAssignment
-          required: true
```

Comment:
? was this updated in the swagger spec?

Reply:
Can you add an IT to cover this case?  That would cover the testing of new generated code

Reply:
Yes, here is the Open API Spec update PR that was approved earlier - https://github.com/okta/openapi/pull/258.